sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -123,6 +123,15 @@ static ipv6_addr_t *_add_addr_to_entry(gnrc_ipv6_netif_t *entry, const ipv6_addr
                 DEBUG("ipv6_netif: error adding prefix to 6LoWPAN-ND management\n");
             }
 #endif
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+            if ((entry->flags & GNRC_IPV6_NETIF_FLAGS_ROUTER) &&
```

Comment:
You even have to. Otherwise the border router will send out two rtr_adv at the same time over the same interface